src/zabapgit_git.prog.abap

```diff
@@ -602,6 +602,8 @@ CLASS lcl_git_pack IMPLEMENTATION.
         rv_type = zif_abapgit_definitions=>gc_type-tree.
       WHEN '011'.
         rv_type = zif_abapgit_definitions=>gc_type-blob.
+      WHEN '100'.
+        rv_type = zif_abapgit_definitions=>gc_type-tag.
```

Comment:
Is this part of changes for annotated tags?

Reply:
Yes, I did some experiments. It's not needed yet. Should I remove it?

Reply:
just keep it, I guess it prevents an error if the repo contains annotated tags